credentialz/gnsi-credentialz.yang

```diff
@@ -48,18 +48,56 @@ module gnsi-credentialz {
         description
             "SSH server credentials freshness-related data.";
 
-        leaf active-trust-bundle-version {
+        leaf active-trusted-user-ca-keys-version {
             type version;
             description
-                "The version of the bundle of the Certificate Authority
-                certificates a.k.a. trust bundle.";
+                "The version of the Certificate Authority keys.";
         }
 
-        leaf active-trust-bundle-created-on {
+        leaf active-trusted-user-ca-keys-created-on {
             type created-on;
             description
-                "The timestamp of the moment when the bundle of the Certificate
-                Authority certificates (a.k.a. trust bundle) was created.";
+                "The timestamp of the moment when the trusted user ca keys
+                 were created.";
+        }
+
+        leaf active-host-certificate-version {
+            type version;
+            description
+                "The version of the host certificate.";
+        }
+
+        leaf active-host-certificate-created-on {
+            type created-on;
+            description
+                "The timestamp of the moment when the host certificate
+                 was created.";
+        }
+
+        leaf active-host-certificate-file-version {
+            type version;
+            description
+                "The version of the host ssh certificate.";
+        }
+
+        leaf active-host-certificate-file-version-created-on {
```

Comment:
Honestly I do not like the idea of 'file' being part of the leaf name. Yes, the key/certificate is in a file (most likely) but what counts is the fact that the key/certificate was created at some point and has the metadata 'version' associated with it. 

Could we get rid of it?

Reply:
Agreed. took out file, PTAL